File: java/program/Clothing.java
class Clothing extends Product{
    //private atributes.
    private int size;
    private String material;
    private char gender;

    /* Constructors */

    public Clothing(){
        this.size = 0;
        this.material = "";
        this.gender = '-';
    }

    public Clothing(int size, String material, char gender){
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    /* Getter and Setter. */

    //get size
    public int getSize(){
        return this.size;
    }

    //set size
    public void setSize(int size){
        this.size = size;
    }

    //get material
    public String getMaterial(){
        return this.material;
    }

    //set material
    public void setMaterial(String material){
        this.material = material;
    }

    //get gender
    public char getGender(){
        return this.gender;
    }
    //set gender
    public void setGender(char gender){
        this.gender = gender;
    }
}

File: php/Clothing.php
<?php

class Clothing{
    private $size = 0;
    private $material = "";
    private $gender = "";

    public function __construct($size = 0, $material = "", $gender = ""){
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    public function setSize($size){
        $this->size = $size;
    }

    public function getSize(){
        return $this->size;
    }

    public function setMaterial($material){
        $this->material = $material;
    }

    public function getMaterial(){
        return $this->material;
    }

    public function setGender($gender){
        $this->gender = $gender;
    }

    public function getGender(){
        return $this->gender;
    }
}

?>